tests/test_psf.py

```diff
@@ -255,5 +255,99 @@ def test_r0_500(self):
+        import copy
+
+        # Make some monochromatic SEDs for testing.
+        sed400 = galsim.SED(
+            galsim.LookupTable(
+                [200,399,400,401,1100], [0,0,1,0,0], interpolant='linear'
+            ),
+            'nm',
+            'fphotons'
+        )
+        sed900 = galsim.SED(
+            galsim.LookupTable(
+                [200,899,900,901,1100], [0,0,1,0,0], interpolant='linear'
+            ),
+            'nm',
+            'fphotons'
+        )
+
+        bandpass = galsim.Bandpass(
+            galsim.LookupTable(
+                [200,201,1099,1100],
+                [0,1,1,0],
+                interpolant='linear'
+            ),
+            wave_type='nm'
+        )
+
+        template = {
+            'psf': {
+                'type': 'AtmosphericPSF'
+            },
+            'input': {
+                'atm_psf': {
+                    'airmass': 1.0,
+                    'rawSeeing': 1.0,
+                    'band':  'r',
+                    'screen_size': 51.2,
+                    'boresight': {
+                        'type': 'RADec',
+                        'ra': { 'type': 'Degrees', 'theta': 0.0},
+                        'dec': { 'type': 'Degrees', 'theta': 0.0}
+                    },
+                    'exponent': None,
+                    'exptime': 600.0,  # Lots of mixing.
+                    'doOpt': False,
+                    '_no2k': True,  # turn off second-kick since it's achromatic
+                }
+            },
+            'image_pos': galsim.PositionD(0,0),  # This would get set appropriately during
+                                                 # normal config processing.
+            'image' : {
+                'random_seed': 1234,
+                'draw_method' : 'phot',
+                'wcs': {
+                    'type' : 'Tan',
+                    'dudx' : 0.2,
+                    'dudy' : 0.,
+                    'dvdx' : 0.,
+                    'dvdy' : 0.2,
+                    'ra' : '@input.atm_psf.boresight.ra',
+                    'dec' : '@input.atm_psf.boresight.dec',
+                },
+                'bandpass': bandpass,
+                'size': 31
+            },
+            'gal': {
+                'type': 'DeltaFunction',
+                'flux': 1e5,
+                'sed': None
+            }
+        }
+
+        for exponent in [-0.2, -0.3]:
+            config400 = copy.deepcopy(template)
+            config400['input']['atm_psf']['exponent'] = exponent
+            config400['gal']['sed'] = sed400
+            img400 = galsim.config.BuildImage(config400)
+            sigma400 = galsim.hsm.FindAdaptiveMom(img400).moments_sigma
+
+            config900 = copy.deepcopy(template)
+            config900['input']['atm_psf']['exponent'] = exponent
+            config900['gal']['sed'] = sed900
+            img900 = galsim.config.BuildImage(config900)
+            img900.wcs = galsim.PixelScale(0.05) # just lie a little bit
```

Comment:
why?

Reply:
Good catch.  That was from an earlier iteration where I was using image.calculateMomentRadius() instead of hsm.  I'll remove it.